.github/workflows/CodeQL Security Scan.yml

```diff
@@ -50,47 +51,29 @@ jobs:
             path: Ed-Fi-ODS-Implementation/
       - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
         working-directory: ./Ed-Fi-ODS-Implementation/
-        shell: pwsh
+        shell: powershell
         run: |
-            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
-            $is_pull_request_branch = 'False'
-            $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
-            if ($is_pull_request_branch -eq $true) {
-              git fetch origin ${{ env.CURRENT_BRANCH }}
-              git checkout ${{ env.CURRENT_BRANCH }}
-            }
-      - name: Checkout Ed-Fi-ODS
+          .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-ODS-Implementation"
+      - name: Checkout Ed-Fi-ODS          
         uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
         with:
             repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
             path: Ed-Fi-ODS/
       - name: Is pull request branch exists in Ed-Fi-ODS
-        working-directory: ./Ed-Fi-ODS/
-        shell: pwsh
+        working-directory: ./Ed-Fi-ODS-Implementation/
```

Comment:
Since we have a build.githubactions.ps1 file in this repo, why not make that the working directory?

Reply:
I changed to this repo